### PR TITLE
refactor/AB#63566_ABC-Replace-Mat-Toolbar-Component

### PR DESCRIPTION
**Pull request by Bricecrx**
# Description

With this ticket, all mat-toolbar call by the application is replaced witha simple native nav call. It has been then restyled to match the former toolbar design. MatToolbarModule calls are also deleted every time it is needed.

NB : mat-toolbar was only called once in the app : in the layout component

# Ticket

I can't paste the azure ticket link because I do not have access to Azure. Here is the ticket id however : [AB#63566](https://dev.azure.com/WHOHQ/2888b816-2d79-49ef-a86e-561e6cd456a0/_workitems/edit/63566)

## Type of change

- [x] Improvement (refactor or addition to existing functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

It has been tested through navigation within the application and verification of replacements.

## Screenshots

image
image
image
image

# Checklist:

( * == Mandatory ) 

- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings

### CODE REVIEW

**Comment by Bricecrx** [comment]
Seen with Antoine : use text-gray-200 and bg-primary-600 as respectively text and background color for now. No need for an adaptation of the text color. (Some changes might be required in the future though)

File: libs/safe/src/lib/components/layout/layout.component.html [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-  </safe-button>
-  <safe-button
-    [isIcon]="true"
-    icon="keyboard_backspace"
-    (click)="goBack()"
-    *ngIf="route"
-    [uiTooltip]="'pages.applications.goTo' | translate"
-  >
-  </safe-button>
+<nav class="flex p-4 items-center md:justify-between text-gray-200 bg-primary-600">
```
Comment (unai-reliefapp) [inline libs/safe/src/lib/components/layout/layout.component.html]: bg-primary should resolve same color as the one set in the `theme.headerColor` right?
@AntoineRelief

Comment (Bricecrx) [reply libs/safe/src/lib/components/layout/layout.component.html]: bg-primary color and theme.headerColor were different. I asked to @AntoineRelief and he told me to use bg-primary

Comment (unai-reliefapp) [reply libs/safe/src/lib/components/layout/layout.component.html]: okay, thanks Brice. Then all good for me

**Pull request merged**